yt/visualization/fixed_resolution_filters.py

```diff
@@ -53,30 +56,19 @@ class FixedResolutionBufferGaussBeamFilter(FixedResolutionBufferFilter):
     """
     This filter convolves
     :class:`yt.visualization.fixed_resolution.FixedResolutionBuffer` with
-    2d gaussian that is 'nbeam' pixels wide and has standard deviation
-    'sigma'.
+    2d gaussian that has standard deviation 'sigma'. Extra arguments
+    are directly passed to `scipy.ndimage.gaussian_filter`.
     """
     _filter_name = 'gauss_beam'
 
-    def __init__(self, nbeam=30, sigma=2.0):
```

Comment:
Instead of removing `nbeam` as a keyword argument, which might cause formerly working scripts to fail, can you deprecate this keyword argument, offering a suggestion for how to emulate it using `ndimage.gaussian_filter` keyword arguments?

Reply:
Sure. How do you raise a deprecation warning?

Reply:
Found it!